test/integration/cluster/cluster_test.go

```diff
@@ -35,7 +35,9 @@ import (
 )
 
 func init() {
-	clusterv1.AddToScheme(scheme.Scheme)
+	if err := clusterv1.AddToScheme(scheme.Scheme); err != nil {
+		panic(err)
```

Comment:
Would it make sense to move this to a `BeforeSuite`, then we could remove `init` and turn this into `Expect(clusterv1.AddToScheme(sheme.Scheme)).To(Succeed())`

Reply:
yeah, that would be fine too, but want the minimal change set so I think that refactor is out of scope for this PR